dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -0,0 +1,903 @@
+<template>
+  <Card class="col-12 page-wrapper-card">
+    <template #title>New Dataset - LkSG </template>
+    <template #content>
+      <div class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
+            v-model="lkSGDataModel"
```

Comment:
What about the list of goods?

Reply:
Ok, it is visible when "is your company a manufacturing company" is set to "yes".

Reply:
Yes I think this is done intentionally by Marcin.